### PR TITLE
Updated startup scripts used in the Auto-scale API and added functionality to send to final model to master instance In sweep()

### LINKED ISSUE #3099: Implement Auto-Scaling on Google Cloud
### Description

In this project, you will implement functionality necessary to automatically spin-up Google cloud machines, load a PyGrid instance, run a training job, and tear down the instance upon completion (depositing the results into another long-running instance). The primary feature will be the ability to run a “hyperparameter sweep”, as exemplified below.

```python
parameters = {“alpha” : [0, 0.01, 0.02], “batch_size” : [32,64]}

gcloud = gr.GoogleCloud(credentials)

cluster = gcloud.LazyCluster(n_machines=10, 
                            type="n-series", 
                            priority="low",
                            eviction_policy="Delete",
                            max_price_usd=0.55,
                            reboot_max_price_usd=0.65)

cluster.sweep(model=model,
             parameters=parameters,
             optim=optim.SGD,
             results_node=grid['my node']
             )
```

### Context

I will be mentoring this project as part of the [Google Summer of Code](https://summerofcode.withgoogle.com/)

Also, this year we have some plans for PyGrid, along with our other projects, including a focus on production: Deployment on Cloud Providers (GCP, Amazon, Azure). You can check out the PyGrid Roadmap [here](https://github.com/OpenMined/Roadmap/tree/master/pygrid_team).

### Required Skills:

- Knowledge of [PyTorch](https://pytorch.org/) and Deep Learning
- Familiarity (or willingness to become familiar) with [PySyft](https://github.com/OpenMined/PySyft)
- Familiarity (or willingness to become familiar) with [Google Cloud’s Infrastructure](https://cloud.google.com/docs)

**Difficulty**: Beginner

While this is a sizeable project, the core functionality is relatively straightforward (provision machines, install and start PyGrid servers, and run training scripts). Furthermore, there are plenty of tutorials on how to run PyGrid servers and how to run Google Cloud machines.

If you are new to PyGrid and want to learn more about it, try out the tutorials. A comprehensive list of tutorials can be found [here](https://github.com/OpenMined/PySyft/tree/master/examples/tutorials/grid). These tutorials cover how to create a PyGrid node and what operations you can perform.

### Useful Links

- PyTorch - https://pytorch.org/blog
- PySyft - https://github.com/OpenMined/PySyft
- PyGrid - https://github.com/OpenMined/PyGrid
- GCP Docs - https://cloud.google.com/docs
- GCloud APIs for Python - https://cloud.google.com/python/docs/reference
- Google APIs (Python Client) - https://github.com/googleapis/google-api-python-client 
- Google Summer of Code - https://summerofcode.withgoogle.com
  - [Timeline](https://developers.google.com/open-source/gsoc/timeline)
  - [Student Guide](https://google.github.io/gsocguides/student/)
  - [OpenMined Org.](https://summerofcode.withgoogle.com/organizations/5561599089180672)

_List of all GSoC project ideas [here](https://docs.google.com/document/d/1OA8277RB9ZN8u7Wgn6NqZK7tzixnZbSlkfulyqoo2G4/edit?usp=sharing)._

### PR DESCRIPTION
**Pull request by rimijoker**
## Description
Updated startup scripts used in the autoscale API to use the latest PyGridNode and PyGridNetwork docker production images.
I have also added a 30sec sleep time at the start of each startup script to avoid docker hub connection issues.
Added functionality to send to final model to master instance In sweep()
I have also updated the README.md for the same.

Fixes #3099
@hericlesme  

## How has this been tested?
- The functionality can be tested using test.py and test.ipynb files.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: syft/grid/utils/autoscale/gcloud.py [hunk 1]
```diff
@@ -241,14 +239,15 @@ def create_cluster(
...
-                sudo -i bash -c 'pip install git+https://github.com/OpenMined/PyGridNode.git'
-                sudo -i bash -c 'echo Starting Node {name} \
-                joined with PyGridNetwork at {pygrid_network_address} & \
-                python -m gridnode --id={name} --port=80 \
-                --gateway_url={pygrid_network_address}'""",
+                sleep 30;
+                docker pull openmined/grid-node:production;
+                docker run \
+                -e NODE_ID={name} \
+                -e HOST="$(curl ifcongif.co)" \
```
Comment (hericlesme) [inline syft/grid/utils/autoscale/gcloud.py:246]: shouldn't be `ifconfig.co`?

Comment (rimijoker) [reply syft/grid/utils/autoscale/gcloud.py:246]: `curl ifconfig.co` gives the external IP, that's what we wanted right?

Comment (hericlesme) [reply syft/grid/utils/autoscale/gcloud.py:246]: What I'm saying is that it's written "ifcongif.co"

Comment (rimijoker) [reply syft/grid/utils/autoscale/gcloud.py:246]: got it 😬

**Review by hericlesme — APPROVED** [review]
